Gruntfile.js

```diff
@@ -87,5 +93,5 @@ module.exports = function (grunt) {
   });
 
   grunt.registerTask('default', ['clean', 'copy:src_to_dist', 'copy:dashboard_to_dist',
-    'copy:examples_to_dist', 'copy:pluginDef', 'babel', 'mochaTest']);
+    'copy:examples_to_dist', 'copy:pluginDef', 'copy:cmsConfig', 'babel', 'mochaTest']);
```

Comment:
smsConfig?

Reply:
The plug-in needs to configure the secret key of the SMS platform and push the mobile phone number, which are stored in the external json file.

Reply:
typo: cms -> sms